src/sentry/templates/sentry/organization-members.html

```diff
@@ -101,7 +109,10 @@
               {% endif %}
             {% endif %}
           </td>
-          <td>{{ member.get_role_display }}</td>
+          <td class="squash">{{ member.get_role_display }}</td>
+          {% feature auth:twofactor %}
+          <td class="squash">{% if has_2fa %}<span style="color: #599748" class="icon-checkmark"></span>{% endif %}</td>
```

Comment:
This color is `@green-dark` from `palette.less`. Not sure if there's a better way to use this color here.